src/utils/networkUtils.ts

```diff
@@ -113,16 +135,16 @@ export const getGasInfo = async () => {
 	}
 };
 
-export function onMetamaskAccountChange(cb) {
+export function onMetamaskAccountChange(cb: () => void) {
 	if (!window.ethereum) return;
 	const listener = throttle(cb, 1000);
 	window.ethereum.on('accountsChanged', listener);
 }
 
-export function onMetamaskNetworkChange(cb) {
+export function onMetamaskNetworkChange(cb: () => void) {
 	if (!window.ethereum) return;
 	const listener = throttle(cb, 1000);
 	window.ethereum.on('networkChanged', listener);
 }
 
-export const isMainNet = (networkId) => networkId === SUPPORTED_NETWORKS.MAINNET;
+export const isMainNet = (networkId: NetworkId) => networkId === 1;
```

Comment:
again, would be good to have Networks as an enum to make equality checks easier

Reply:
happy to replace this when we switch to the new library (which has the networks, etc)